xarray/coding/variables.py

```diff
@@ -309,6 +309,7 @@ def encode(self, variable: Variable, name: T_Name = None):
         dtype = np.dtype(encoding.get("dtype", data.dtype))
         fv = encoding.get("_FillValue")
         mv = encoding.get("missing_value")
+        unsigned = encoding.get("_Unsigned") is not None
```

Comment:
Thanks @kmuehlbauer can you add a comment describing the unsigned behaviour please?